src/MAUI/Maui.Samples/Views/ApiKeyView.xaml

```diff
@@ -1,45 +1,37 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
-<ContentView
-    x:Class="ArcGIS.ApiKeyView"
-    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
-    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+<ContentView x:Class="ArcGIS.ApiKeyView"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
     <ContentView.Content>
         <StackLayout Orientation="Vertical">
-            <Label x:Name="Instructions" Margin="5">
-                <Label.FormattedText>
-                    <FormattedString>
-                        <Span Text="Some content used in the samples requires a developer API key for access. Go to " />
-                        <Span
-                            Text="the developer documentation"
-                            TextColor="Blue"
-                            TextDecorations="Underline">
-                            <Span.GestureRecognizers>
-                                <TapGestureRecognizer Tapped="LinkTapped" />
-                            </Span.GestureRecognizers>
-                        </Span>
-                        <Span Text=" to learn how to obtain a developer API key for ArcGIS Online." />
-                    </FormattedString>
-                </Label.FormattedText>
-            </Label>
-            <StackLayout Orientation="Horizontal">
+            <HorizontalStackLayout Margin="5">
+                <Label Text="Some content used in the samples requires an API key to access ArcGIS location services. Go to " />
```

Comment:
Does this require a url here? such as the url to the create an API Key tutorial?
as below
```
Go to <Hyperlink NavigateUri="https://links.esri.com/create-an-api-key" RequestNavigate="Hyperlink_RequestNavigate">
                the developer documentation tutorial
```

Reply:
MAUI works a bit differently to WPF and WinUI there is a tap gesture event

```
                    <Label.GestureRecognizers>
                        <TapGestureRecognizer Tapped="LinkTapped" />
                    </Label.GestureRecognizers>
```

Which calls

```
        private void LinkTapped(object sender, TappedEventArgs e)
        {
            Microsoft.Maui.ApplicationModel.Launcher.OpenAsync(new Uri("https://links.esri.com/create-an-api-key"));
        }
```

So it should have the same workflow as the other frameworks and take a user to the tutorial correctly.

Reply:
Thanks for explaining that.